File: Catalog core/src/main/java/cz/vhromada/catalog/repository/ProgramRepository.java
package cz.vhromada.catalog.repository;

import cz.vhromada.catalog.dao.entities.Program;

import org.springframework.data.jpa.repository.JpaRepository;

/**
 * An interface represents repository for programs.
 *
 * @author Vladimir Hromada
 */
public interface ProgramRepository extends JpaRepository<Program, Integer> {
}

File: src/main/kotlin/cz/vhromada/catalog/repository/ProgramRepository.kt
package cz.vhromada.catalog.repository

import cz.vhromada.catalog.domain.Program
import org.springframework.data.jpa.repository.JpaRepository

/**
 * An interface represents repository for programs.
 *
 * @author Vladimir Hromada
 */
interface ProgramRepository : JpaRepository<Program, Int>
